project/src/helpers.ts

```diff
@@ -1,7 +1,7 @@
 
 import { FavoritesByCity, Offer } from './types/offer';
 
-export const getOffersByCity = (offers: Offer[], activeCity: string) => offers.filter(({ city }) => city.name === activeCity) ?? [];
+export const loadOffersByCity = (offers: Offer[], activeCity: string) => offers.filter(({ city }) => city.name === activeCity) ?? [];
```

Comment:
```suggestion
export const loadOffersByCity = (offers: Offer[], activeCity: string) => offers.filter(({ city }) => city.name === activeCity);
```
фильтр всегда вернет массив